osu.Game/Database/RealmExtensions.cs

```diff
@@ -1,51 +1,26 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using System.Collections.Generic;
-using AutoMapper;
-using osu.Game.Input.Bindings;
+using System;
 using Realms;
 
 namespace osu.Game.Database
 {
     public static class RealmExtensions
     {
-        private static readonly IMapper mapper = new MapperConfiguration(c =>
+        public static void Write(this Realm realm, Action<Realm> function)
```

Comment:
What are these to be used for? They're unused as far as the scope of this pull goes. There is one `realm.Write()` call in `KeyBindingRow` but that one uses an actual proper method defined on `Realm` rather than these extensions.

Reply:
https://github.com/realm/realm-dotnet/issues/2660 should sum it up. i think it will end up being quite a common use case.